test-files/jest-errors.test.ts

```
@@ -0,0 +1,88 @@
+    expect(obj).toBe({ a: 1, b: 2 })
+  })
+  
+  // Not using async/await properly
+  it('async test error', () => {
+    return Promise.resolve(42).then((result: number) => {
+      expect(result).toBe(42)
+    })
+  })
+  
+  // Using done callback incorrectly
+  it('done callback error', (done) => {
```

ESLint

[error] Return a Promise instead of relying on callback parameter

(jest/no-done-callback)